src/app/utils/badge-color.ts

```diff
@@ -0,0 +1,26 @@
+import { getTheme } from 'office-ui-fabric-react';
+
+export function getMethodStyle (method: string) {
+  const currentTheme = getTheme();
+
+  method = method.toString().toUpperCase();
```

Comment:
Type of method is String. Why do we need to convert it to a string before uppercasing it?

Reply:
Good catch